### PR TITLE
style: update wallet connect modal

### LINKED ISSUE #7319: Remove shadow from wallet consent button
image
- to keep it consistent with our design language, I would remove the drop shadow from the button. 

Please use correct padding for the modal here as well: 
image
- should be 32px horizontal and match the modal heading padding and alignment

### PR DESCRIPTION
**Pull request by floyd-li**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #7319
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=13rv1SWoLg9Gb3tmvHPZxb7JbVy51BtMziX7k9WQGSJ7Kp3A)

#### Community participation

- [x] [Are you at KodaDot Ecosystem Telegram?](https://t.me/kodadot_eco)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

### <samp>🤖 Generated by Copilot at 589781c</samp>

Refactored and enhanced the UI of the `ConnectWallet` and `NeoButton` components. Fixed some styling issues and made the components more reusable.

### <samp>🤖 Generated by Copilot at 589781c</samp>

> _We're the coders of the sea, we make the web look fine_
> _We tweak the `NeoButton` and the modal's align_
> _We heave and ho and push and pull, we work as a team_
> _We sing this shanty loud and proud, we're living the dream_

### CODE REVIEW

**Comment by exezbcz** [comment]
fast! 

could you please change these two as well please, other than that, good


image
image

File: styles/components/_connect-wallet.scss [hunk 1]
```diff
@@ -38,7 +38,7 @@
     }
 
     .more-option-button {
-      margin: 0 34px;
+      margin: 0 2rem;
```
Comment (Jarsen136) [inline styles/components/_connect-wallet.scss:41]: `mx-0 my-6`

**Review by exezbcz — APPROVED** [review]

**Review by Jarsen136 — APPROVED** [review]

**Comment by yangwao** [comment]
pay 20 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $20 @ 4.04 USD/DOT ~ 4.95 $DOT
🧗 [13rv1SWoLg9Gb3tmvHPZxb7JbVy51BtMziX7k9WQGSJ7Kp3A](https://polkadot.subscan.io/account/13rv1SWoLg9Gb3tmvHPZxb7JbVy51BtMziX7k9WQGSJ7Kp3A)
🔗 [0x878eddb82318d34c7e7a4bce07bac0c9d889a67b9b59deebf0c8a4beefb1999e](https://polkadot.subscan.io/extrinsic/0x878eddb82318d34c7e7a4bce07bac0c9d889a67b9b59deebf0c8a4beefb1999e)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**